### DISCUSSION TITLE
F10 (disable mouse buttons) doesn't disable mouse side buttons during gameplay.

### DISCUSSION THREAD

**Discussion by Siackerz** [ppy/osu#21876] [Q&A] [answered]
I have a weird mouse grip so my thumb sometimes hits the back mouse button and it pauses gameplay during a play. 

Is there a reason on why the side mouse buttons aren't disabled when you have mouse buttons disabled? 
(this doesn't exist in standard because disabling mouse buttons disables the side back button) 

example: 

https://user-images.githubusercontent.com/78198710/209785857-728645b2-c1f7-4421-81a3-5e00ff855055.mov

**Comment by bdach** [comment, marked as answer]
That's because the "back" action in keybindings is bound to mouse4 by default.

As a workaround you can unbind this. I'm unsure whether that's something that should be covered by the mouse button disable, though.
